### ISSUE TITLE
Lifetime stats missing

### ISSUE THREAD

**Issue by eXoticD** [ValveSoftware/Dota2-Gameplay#12387]
### Description

Hey

Where can I find stats of Rampages, Ageis steals, and all other stats?

there are no + to open this menu after the update

**Comment by martymartz** [comment]
yes this post needs to be addressed valve i got 2 rampages since the update is it even still recorded? do you know how hard it is to get a rampage valve ? its very hard so we would appreciate it if you fix this terrible MISSING FEATURE!!

**Comment by martymartz** [comment]
also most successful heroes are missing
